src/utilities/text/text.css

```diff
@@ -17,3 +17,7 @@
 .text-transform--uppercase {
   text-transform: uppercase;
 }
+
+.lineheight-m {
```

Comment:
bliver denne class brugt?

Reply:
Ja, den blev brugt lidt rundt omkring i eb, ved navn lh-normal. Den er nu ændret til lineheight-m for et mere passende navn, hvis det en dag giver mening at udvide lineheight-s/l/xl.